### PR TITLE
API Keys: Documentation

### PR DESCRIPTION
**Pull request by krivard**
Partially addresses AK-59

Draft mode until we know what the rate limit is

[Rendered preview here](https://krivard.github.io/delphi-epidata/api/privacy_statement.html)

### Summary:

API Keys comms:
* Home page
* API Keys overview page, including usage instructions
  * I included this as a sidebar header just after the home page, so all the other toplevel pages got bumped down
* Privacy statement

### Prerequisites:

- [x] Merging against `api-keys`
- [ ] Branch is up-to-date with the branch to be merged with
- [x] Build is successful
- [x] Code is cleaned up and formatted

### CODE REVIEW

File: docs/api/api_keys.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+has_children: true
+---
+
+# Epidata API Keys
+
+Anyone may access the Epidata API anonymously without providing any personal
+data. Anonymous API access is subject to the following restrictions:
+
+1. public datasets only
+1. rate-limited to $RATE_LIMIT
```
Comment (krivard) [inline docs/api/api_keys.md:13]:
```suggestion
1. rate-limited to 100 requests per hour
```

File: docs/index.md [hunk 2]
```diff
@@ -14,6 +14,25 @@ group](https://delphi.cmu.edu/). The Epidata API includes:
...
+
+If you regularly or frequently use our system, please consider using an API key
+even if your usage falls within the anonymous usage limits. API key usage helps
+us understand who and how others are using our Delphi Epidata API, which may in
+turn inform our future research, data partnerships, and funding.
+
+For more information about how we use the data you provide us through your
+registration and API request activity, see our [Privacy
+Statement](api/privacy_statement.md). At any time, you may submit a [Deletion
+Request](https://forms.gle/GucFmZHTMgEFjH197) to have us deactivate your key and destroy all
```
Comment (krivard) [inline docs/index.md:34]:
```suggestion
registration and API request activity, see our 
[Privacy Statement](api/privacy_statement.md). At any time, you may submit a 
[Deletion Request](https://forms.gle/GucFmZHTMgEFjH197) to have us deactivate your key and destroy all
```

**Review by krivard — COMMENTED** [review]
update rate limit and fix link formatting

**Review by melange396 — COMMENTED** [review]
looks pretty good!  does it matter that `docs/api/covidcast.md` and `docs/api/api_keys.md` both have `nav_order: 2`?

File: docs/api/api_keys.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+title: API Keys
+nav_order: 2
```
Comment (krivard) [inline docs/api/api_keys.md:3]:
```suggestion
nav_order: 1
```

**Comment by krivard** [comment]
@melange396 good catch!

**Review by melange396 — APPROVED** [review]

**Pull request merged**